hw2/02-search.js

```diff
@@ -1,3 +1,62 @@
-// Add your code here
+const searchButton = document.querySelector("input[type='button']");
+const userInput = document.getElementById('userInput');
+const cardsContainer = document.getElementById('cardDeck');
 
-userInput.addEventListener('click', handleClick);
+function handleSearch() {
+  const keyword = userInput.value.toLowerCase().trim();
```

Comment:
Everything works perfectly. Only thing I noticed was if you search with empty text every character gets returned. If you didn't want to return every card, you could check first if the search bar text is empty.